cartographer/mapping/internal/3d/scan_matching/fast_correlative_scan_matcher_3d_test.cc

```diff
@@ -120,7 +116,11 @@ class FastCorrelativeScanMatcher3DTest : public ::testing::Test {
                                 {},
                                 point_cloud_,
                                 low_resolution_point_cloud,
-                                Eigen::VectorXf::Zero(10)};
+                                GetRotationalScanMatcherHstogram()};
+  }
+
+  static Eigen::VectorXf GetRotationalScanMatcherHstogram() {
+    return Eigen::VectorXf::Zero(10);
```

Comment:
how do the tests work if the histogram used to build the matcher only has zeros? (i know you didn't change that, just wondering)

Reply:
In these tests the `FastCorrelativeScanMatcher` is initialized with a histogram containing just zeros. The tests call `CreateConstantData` which creates a `TrajectoryNode` with a histogram containing just zeros. So matching the histograms is quite trivial in this case.